contribute.html

```diff
@@ -0,0 +1,271 @@
+    
+            <div class="row text-center" style="justify-content: center">
+                <div class="col-sm-12 col-md-6">
+                    <a class="card-2" href="https://loopback.io/doc/en/contrib/doc-contrib.html" target="_blank">
+                    <div class="row">
+                        <div class="col-sm-3">
+                            <img class="icon" src="img/contribute/contribute-icon-1x.png" alt="Chat room icon"/>
+                        </div>
+                        <div class="col-sm-9">
+                            <h5 class="card-title">Contribute to documentation</h5>
+                            <p class="regular text-gray">We welcome input and help with documentation. 
+                                Check out the documentation sources.</p>
+                        </div>
+                    </div>
+                    </a>
+                </div>
+        
+                <div class="col-sm-12 col-md-6">
+                    <a class="card-2" href="https://groups.google.com/forum/#!forum/loopbackjs" target="_blank">
+                    <div class="row">
+                        <div class="col-sm-3">
+                            <img class="icon" src="img/contribute/devchat-icon-1x.png" alt="Contribute icon"/>
+                        </div>
+                        <div class="col-sm-9">
+                            <h5 class="card-title">Participate in the community</h5>
+                            <p class="regular text-gray">The LoopBack community carries on lively discussions in several places. 
+                                Feel free to learn and participate.</p>
+                        </div>
+                    </div>
+                    </a>
+                </div>
+            </div>
+        </div>
+    </section>
+    <hr class="hairline">
+
+    <!-- Governance section -->
+    <section class="even-section">
+        <div class="container">
+            <h3 class="section-heading text-center" style="padding-top: 48px;">Governance</h3>
+            <p class="regular text-center">The LoopBack project governance model follows the sprint and tradition of open source.</p>
+            <p class="large" style="padding-left:190px;padding-right:190px;padding-top:32px;padding-bottom:24px;">
+                LoopBack is an open, inclusive, and tolerant community of people working together to build 
+                a world-class Node framework and tools. We value diversity of individuals and opinions, and seek to operate on
+                consensus whenever possible. We strive to maintain a welcoming, inclusive, and harassment-free environment, 
+                regardless of the form of communication. When consensus is not achievable, we defer to the owners of each 
+                individual module; the powers of the individual owner are kept in check by the ability of the community to 
+                fork and replace dependencies on the individual module and maintainer.</p>
```

Comment:
i'm thinking whether we include https://loopback.io/doc/en/contrib/Governance.html as "see more details". 
@jannyHou @nabdelgadir , what do you think?

Reply:
<img width="940" alt="screen shot 2018-12-13 at 8 56 00 am" src="https://user-images.githubusercontent.com/42985749/49943070-f2732180-feb4-11e8-8170-8146d54e55d7.png">
Something like this?